app/templates/_package.json

```diff
@@ -4,7 +4,7 @@
   "description": "<%= moduleDesc %>",
   "main": "index.es5.js",
   "scripts": {
-    "coverage": "isparta cover _mocha index.js -- --require babel/register",
+    "coverage": "babel-node node_modules/.bin/isparta cover _mocha",
```

Comment:
have you seen https://github.com/ambitioninc/babel-istanbul? maybe its more usefull and not that crappy as isparta